common/alerts.js

```diff
@@ -52,5 +52,21 @@
                 };
             }
         };
-    }]);
+    }])
+    .directive('compile', ['$compile', function ($compile) {
+      return function(scope, element, attrs) {
```

Comment:
To follow the way we have defined each of our directives, you should change the `return` to return an object:
```
return {
  restrict: `A`,     // could also be `E` (element), `C` (class), `M` (comment`
  link: function(scope, element, attrs) {
    ...
  }
}
```